thrifty-integration-tests/bar/Bar.thrift

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Let's stick with the Microsoft copyright header; this Apache Foundation header is on a few specific .thrift files that we have taken from the Apache implementation, but new files should not be thusly attributed.